editor/plugins/path_2d_editor_plugin.h

```diff
@@ -34,35 +34,39 @@
 #include "editor/editor_plugin.h"
 #include "scene/2d/path_2d.h"
 #include "scene/gui/box_container.h"
+#include "scene/gui/dialogs.h"
 
 class CanvasItemEditor;
 class MenuButton;
 
 class Path2DEditor : public HBoxContainer {
 	GDCLASS(Path2DEditor, HBoxContainer);
 
+	friend class Path2DEditorPlugin;
 	CanvasItemEditor *canvas_item_editor = nullptr;
 	Panel *panel = nullptr;
 	Path2D *node = nullptr;
 
-	HBoxContainer *base_hb = nullptr;
-
 	enum Mode {
 		MODE_CREATE,
 		MODE_EDIT,
 		MODE_EDIT_CURVE,
 		MODE_DELETE,
-		ACTION_CLOSE
+		MODE_CLOSE,
+		MODE_CLEAR_POINTS
```

Comment:
```suggestion
		MODE_CLEAR_POINTS,
```
More friendly for adding more modes.